### PR TITLE
Feature: flow revision status

### PR DESCRIPTION
**Pull request by anshulrr**
1. Flow revision status column
2. Seed update
3. Set status to done after creation of flow
4. Graphql API doneEditFlow

### CODE REVIEW

File: api.docs/includes/_flows.md [hunk 1]
```diff
@@ -359,6 +359,66 @@ Type | Description
 --------- | ---- | ------- | -----------
 <a href="#flowresult">FlowResult</a> | An error object or empty
 
+## Done Edit a Flow
+
+```graphql
+mutation doneEditFlow($id: ID!) {
```
Comment (pankaj-ag) [inline api.docs/includes/_flows.md]: This looks a bit confusing. Can we rename this API endpoint something like **publish_flow**

File: lib/glific/flows.ex [hunk 2]
```diff
@@ -87,7 +87,8 @@ defmodule Glific.Flows do
       {:ok, _} =
         FlowRevision.create_flow_revision(%{
           definition: FlowRevision.default_definition(flow),
-          flow_id: flow.id
+          flow_id: flow.id,
+          status: "done"
```
Comment (pankaj-ag) [inline lib/glific/flows.ex]: Default status should be draft right?

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: lib/glific_web/resolvers/flows.ex [hunk 3]
```diff
@@ -64,4 +64,14 @@ defmodule GlificWeb.Resolvers.Flows do
       {:ok, flow}
     end
   end
+
+  @doc false
+  @spec done_edit_flow(Absinthe.Resolution.t(), %{id: integer}, %{context: map()}) ::
+          {:ok, any} | {:error, any}
+  def done_edit_flow(_, %{id: id}, _) do
```
Comment (pankaj-ag) [inline lib/glific_web/resolvers/flows.ex]:
```suggestion
  def publish(_, %{id: id}, _) do
```

**Review by pankaj-ag — CHANGES_REQUESTED** [review]

File: lib/glific/flows.ex (hunk 2 above)
Comment (anshulrr) [reply lib/glific/flows.ex]: We might need to decide later, whether a flow should have at least one revision with status done.

File: api.docs/includes/_flows.md (hunk 1 above)
Comment (anshulrr) [reply api.docs/includes/_flows.md]: Yes, that makes sense.

**Review by pankaj-ag — APPROVED** [review]

**Pull request merged**